src/warnet/control.py

```diff
@@ -214,25 +217,24 @@ def run(scenario_file: str, additional_args: tuple[str]):
                 "containers": [
                     {
                         "name": name,
-                        "image": "bitcoindevproject/warnet-commander:latest",
-                        "args": additional_args,
+                        "image": "bitcoindevproject/warnet-commander:dev",
```

Comment:
Before merging this line needs to be reverted (and a new image pushed)

Reply:
this kind of thing is so awkward. i know its a LOT but i feel like ideally CI would build images and use them locally, then on merge to main the image gets pushed to dockerhub. But that means that in CI we'd want to "never" pull (use local image) and in prod we'd want to "always" pull (?) to make sure the latest is used?

Reply:
+1, I think our CI/image pipeline could use some love. Don't think it should be a blocker here and not needed before Tabconf, so I created a backlog to track https://github.com/bitcoin-dev-project/warnet/issues/520